docs/001_develop/03_client-capabilities/009_presentation/connection-indicator.mdx

```diff
@@ -14,13 +14,13 @@ import ConnectionIndicatorDemo from './examples/connection-indicator.js';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# ConnectionIndicator
+The ConnectionIndicator component provides a clear visual indicator of the status of the connection between the user device and the app server.
 
 Use cases:
-* Visual representation of the websocket connection status.
+* visual representation of the websocket connection status
 
 ## Example
-
+The example below shows a communication indicator, which should be set to `disconnected`.
```

Comment:
I feel what you've written could be confusing that the user is required to set it to disconnected themselves, but I could be wrong
```suggestion
The example below shows a communication indicator, in this case showing `disconnected`.
```